Dockerfile-windows

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2015-2019 Tigera, Inc. All rights reserved.
```

Comment:
The dockerfile-windows is named this way due to the way the makefile loads arches based on docker files: https://github.com/projectcalico/go-build/blob/296c6f3239d6a1f42c691c5b09c9d36f0f617107/Makefile.common#L24

It was tricky to figure out what going on here. There is a probably a better way to do this?

Reply:
Yeah... this is something we'll need to figure out. Apart from the dockerfile arches thing, right now we don't have a good way to build Windows images because Semaphore CI doesn't support Windows yet.

Reply:
since these are mostly executable it's easy enough to build them on Linux using buildx but they can only be exported to a registry, not to the file system.